src/zoktai/any.md

```diff
@@ -2,55 +2,67 @@
 
 ## About this guide
 
+This guide assumes you are playing on the japanese 1.0 release of Boktai 2.  
+It will likely work fine for other releases but some strategies like RNG manipulation will need different timings.  
 Directions for paths and exits will use ↖️, ↗️, ↙️ and ↘️.
 
+### Terminology
+
+- RNG manip: Short for "Random Number Generator Manipulation". Used to influence the randomness within the game to make things happen that are normally reliant on chance.
+- Soft reset: Pressing A + B + Start & Select at the same time reboots the game. This is useful for RNG manips and is faster when loading a save after a death.
+- STRG and AGIL: Strength and Agility. The second and third status point options. They influence your physical damage, movement speed and defense.
+
 ## Before starting
 
-If playing on original hardware with an existing save file, skip ahead to "Initial Game Settings".  
-If playing on BizHawk emulator, follow the remaining instructions in this section before starting a run.
+Make sure you have an existing save file and that the Crossover option is unlocked in the Link menu.  
+To unlock the Crossover option, go to the Link menu, then press the following buttons in order:  
+`L R L R L L R R R R L L Select Start Select Start`.
+
+### Emulator settings
 
-Set Skip BIOS to FALSE and RTC to FALSE in the GBA Settings (Requires v2.6.3+):
-<img class="center" src="./assets/images/emu_settings.webp">
+- Requires BizHawk 2.6.3 or newer.
+- Set "Skip BIOS" and "RTC" to False in the GBA Settings.
 
-Start a new game and create a save file after killing the tutorial Zombie with the Sol de Vice.  
-Create a save file (This is required to access the Link menu) and soft-reset (A+B+Start+Select).
+Setting "Skip BIOS" is mandatory as this creates parity with actual hardware.  
+Disabling RTC is optional but recommended since it simulates an empty cartridge battery.
```

Comment:
Recommend not describing RTC as optional, having control of the ingame clock is integral to forcing the coffins to teleport.